File: lib/countries.js
/** Parse the countries.html file downloaded from the fco site */

fs = require('fs')
fs.readFile('../data/countries.html', 'utf8', function (err,data) {
  if (err) {
    return console.log(err);
  }
  
  var pattern = /<a href="(.*)">(.*)<\/a>/g;
  var link = pattern.exec(data);
  console.log(link);
});

File: lib/countries.rb
file = File.new('../data/countries.html')
print "[\n"
while (line = file.gets)
  m =  /<a href="([^"]*)">([^<]*)<.a>/.match(line)
  if (m) 
    print "{ 'name': '" + m[2] + "', 'link': '" + m[1] + "'},\n"
  end
end
print "]\n"